js/src/redux/providers/registry/middleware.js

```diff
@@ -14,13 +14,40 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
+import { debounce } from 'lodash';
 import Contracts from '~/contracts';
 import subscribeToEvents from '~/util/subscribe-to-events';
 
 import registryABI from '~/contracts/abi/registry.json';
 
 import { setReverse, startCachingReverses } from './actions';
 
+const read = (chain) => {
+  const reverses = window.localStorage.getItem(`${chain}-registry-reverses`);
```

Comment:
1. Would prefer using `store` (i.e. `import store from 'store';`)
2. Would prefer the @ngotchac introduced `_parity::` prefix (still need to migrate old)
3. Would prefer `::` seperators as per above convention
4. Would prefer `<prefix>::<type>::<subtype>`, i.e. `_partity::reverse::${chain}`